web/assets/styles/common.scss

```diff
@@ -1,6 +1,7 @@
 html,
 body {
-    background: var(--body-bg);
+    background: #000;
+    background: var(--body-bg, #000);
```

Comment:
This sort of trick doesn't work, because of SASS :(
The compiler discards the first line, leaving only the second one which won't be interpreted by older browsers.
Maybe there's a way in SASS to force both lines to appear:
https://stackoverflow.com/questions/44271920/css-variables-with-fallback-for-older-browsers
(have to wrap my head around this)